install.rst

```diff
@@ -26,11 +26,12 @@ FreeBSD and OpenBSD, or build Zeek yourself.
 Docker Images
 =============
 
-We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeek
+We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeek.
 
     * For the latest feature release: ``docker pull zeek/zeek:latest``
     * For the latest LTS release: ``docker pull zeek/zeek:lts``
```

Comment:
Should we remove `lts` from here even if we're still pushing it?

Reply:
I wouldn't, seems potentially useful for others to build on in their own CI pipelines, for ad-hoc testing, etc. I think this is a little different from the binary images, which are more likely to end up in production use than these images. We could add a note similar to the one for binary images if you're that uneasy about it.